### ISSUE TITLE
Invitation component

### ISSUE THREAD

**Issue by ddfridley** [EnCiv/undebate-ssp#50]
```
<Invitation
    className={}
    style={}
    electionOM={[electionObj, electionMethods}
}
```
See #46 for electionOM details

- [x] `Submit Invite` should be a separate component, to which electionOM is passed.  For now create an app/submit-invite.js component that renders the button - and the rest will be done in another issue.
- [x] support message field seen in the figma

Prerequisites: 
- #9 

Figma: https://www.figma.com/file/yXrcMvg5EIwdLkHHxIdILh/Undebate-Self-Service-Portal?node-id=344%3A5652

- [x] See designer's comments to the right of the figma component drawing

![image](https://user-images.githubusercontent.com/3317487/144123750-8dfb85ff-1496-4039-a4b5-85570217c81a.png)

**Comment by ddfridley** [comment]
@kamui-fin When the user clicks on Send Invite, it should do this:
```
emit('send-invite', electionObj._id, success=>{...))
```
The last parameter is a callback where the server acknowledges that the request has been received, but you don't know it's been sent until the data sent gets updated. 

- [ ] The button should be inactive (gray) until the moderator name and email have been entered and upserted. 
- [ ] See the discussion at https://www.figma.com/file/yXrcMvg5EIwdLkHHxIdILh?node-id=344:5615#132441050 about the date list
- [ ] See #46 for the moderator invitation sentDate but right now (12/11) it's only one date and we need to make a list so this structure should change soon.

**Comment by kamui-fin** [comment]
Is this what you mean by displaying the sent date? 
![image](https://user-images.githubusercontent.com/65237048/145733848-3f1c65b8-1b59-4e41-b292-4c513ffb5064.png)

Also, how would the user be transferred to the Election Table section if the button turns into a "Resend Invite" button after the user has clicked it?

**Comment by ddfridley** [comment]
All I was thinking about was a list of dates that invites were sent at. But @procupid might think of better ways to display that. 
I think that when the button says send invite, after the user clicks on it it goes to the next page.
But if the user comes back to this page it will say resend invite.
Or - we change the UI to have a Next button and a Send Invite button. But for now I'm going for the simplest.  Mostly lets get this built and then we can get feedback and iterate.

**Comment by ddfridley** [comment]
this was closed by PR #68
